ios/PointSdkRn+DataManager.swift

```diff
@@ -212,4 +212,24 @@ import PointSDK
       }
     }
   }
+
+    @objc
+    func getInsights(_ params: [String: Any], resolve:  @escaping RCTPromiseResolveBlock, rejecter reject: @escaping RCTPromiseRejectBlock) -> Void {
+        Task {
+            do {
+                let startDate = (params["from"] as? String)?.fromIsoStringToDate()
+                let endDate = (params["to"] as? String)?.fromIsoStringToDate()
+                let offset = params["offset"] as? Int
+                guard let typesStrings = params["types"] as? [String], typesStrings.count > 0 else {
+                    resolve([])
```

Comment:
since the type is mandatory i believe we should use the reject call with a message instead of resolve